features/document-viewing.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Document viewing
+  As a specialist member of the public
```

Comment:
Since you're changing it, this is as good a time as any to update it. We've generally been referring to users with specialist needs rather than specialist users. So "As a member of the public with specialist needs"